Resources/config/form-type.xml

```diff
@@ -9,5 +9,13 @@
             <tag name="form.type" alias="cmf_core_checkbox_url_label" />
             <argument type="service" id="router" />
         </service>
+
+        <service id="cmf_core.form.extension.publishable" abstract="true" class="Symfony\Cmf\Bundle\CoreBundle\Form\Extension\PublishableExtension">
+            <argument></argument><!-- Must be overrided when using this service as parent -->
```

Comment:
well, for cmf 2.0 we can imho depend on the newest stable symfony version if we want. get the most out of it rather than stay stuck on legacy support. but does not need in this PR.

@wouterj what is your opinion on this topic?